examples/avmnist_multimodal/main.py

```diff
@@ -28,7 +28,7 @@ def arg_parse():
     parser.add_argument(
         "--gpus",
         default=0,
-        help="gpu id(s) to use. None/int(0) for cpu. list[x,y] for xth, yth GPU."
+        help="gpu id(s) to use. int(0) for cpu. list[x,y] for xth, yth GPU."
         "str(x) for the first x GPUs. str(-1)/int(-1) for all available GPUs",
```

Comment:
Why does `dafault=0` here and `dafault=1` in other examples, explain `default=0`?

Reply:
It is set to 0 originally here, i.e. using CPU only. I did not change it due to my unfamiliarity with this example.